server/trips/trips.js

```diff
@@ -68,6 +68,19 @@ const getTripInfo = trip =>
     })
   .catch(err => err));
 
+Trip.addFriendsToTrip = (id, friends) =>
+  Trip.findOne({ where: { id } })
```

Comment:
Aren't these `friends` user objects from the db? Can we just do `trip.addUsers(friends)`?